packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -89,6 +93,9 @@ const TradingAppCard = ({
         else;
     };
 
+    const [is_modal_open, setModalOpen] = useState(false);
```

Comment:
we are following `React.useState` usage style.

Reply:
Ahh okay. Any specific reason we are not importing and using it?